cosmos/test/test_legacy.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from legacy.models import AuthUser, MysiteProfile
+from legacy.tokens import account_import_token
+
+
+class LegacyTestCase(TestCase):
+    databases = {"default", "legacy"}
```

Comment:
Forgot to add this in my review, but pls add the following tests:

- user sucessfully migrated
- user already migrated
- user migrating with different emails
  - success when - `@student.tue.nl`, `@tue.nl`
  - fail when - `@cosmostue.nl`, `@gmail.com`